style/fontsizes.less

```diff
@@ -0,0 +1,26 @@
+@standard: 14px, 15px, 16px, 17px, 18px, 19px, 20px;
+@title: 22px, 23px, 24px, 25px, 26px, 27px, 28px;
+@preview-title: 15px, 16px, 17px, 18px, 19px, 20px, 21px;
+@desc: 12px, 13px, 14px, 15px, 16px, 17px, 18px;
+
+each(range(7), {
+	.font-size-@{value} {
+		font-size: extract(@standard, @value);
+
+		&.title {
```

Comment:
The font-size is global so you could apply it once, higher up, instead of adding it to every component.

As it stands, the knowledge of which components have adjustable font-size is duplicated to both the components themselves and here. Only one of those is actually needed.

Reply:
Hmm... once you see the redundancy, you can't unsee it. How about https://github.com/wikimedia/wikipedia-kaios/pull/300/commits/1c945b4d5b7bec571883c0528cf6cd7275b264cd? Using `document.body.className` instead of `classList.add` takes over the `body` class name, I do this to replace font size class as it changes (instead of adding more, `font-size-1 font-size-2` etc). If we ever wanted to another class to the `body` for some reason, we would have to take a look at this part